src/prefix.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import canUseDOM from 'can-use-dom';
+
+let prefix;
+
+if (canUseDOM) {
+    // Client-side, we can use the dynamic prefixer to avoid unnecessary work.
+    const Prefixer = require('inline-style-prefixer');
+
+    // We need keepUnprefixed so our sorting code knows how to order the
+    // prefixed styles.
+    const prefixer = new Prefixer({
```

Comment:
:( That's a pretty big size bump. I'd like to get this landed because of the bug fix in it, but I'm worried about throwing this significant change onto users. Could we maybe pull this change out into a separate PR to discuss?